### ISSUE TITLE
Week 3 Day 2 Ex4 Optimize. 

### ISSUE THREAD

**Issue by eslopfer** [01-edu/Branch-AI#34]
sklearn is not included in the libraries when setting up the environment in ex00. 

Also for question2 the epoch 50 accuracy in my case is 90%, and checking Lee’s it's 85%, with the same code. Perhaps allow more margin for randomness?

**Comment by brad-gh** [comment]
Yes, there's a random selection of the points to calculate the gradient. I'll put 80% and let's see how the students perform, I'll re-run the code in the meantime.

**Comment by eslopfer** [comment]
[day12ex04.zip](https://github.com/01-edu/Branch-AI/files/9837658/day12ex04.zip)
Hi I attached the notebook. @brad-gh as far as my limited domain knowledge goes, I think it's being scaled could you kindly take a look?
